ui/00_message/common/item_name.xml

```diff
@@ -19659,7 +19659,7 @@
   </item>
   <item id="7839">
     <original>鬼どぶろく</original>
-    <translation>Demon Unrefined Wine</translation>
+    <translation>Unrefined Demon Wine</translation>
```

Comment:
From what I gather, it's referring to unfiltered/unrefined demon sake
The literal translation is oni doburoku; i don't think i need to explain oni here.
Doburoku is an unrefined sake that can easily be made at home from steamed rice, kōji, water and yeast. Creation of this sake was outlawed during the meiji period apparently.

Apart from this, grammatically, i think both of these are correct but Unrefined Demon Wine sounds better. It's unrefined "wine"(sake) made by demons.